components/alert/alert.css

```diff
@@ -0,0 +1,31 @@
+.cs-alert {
+    user-select: none;
+    border: 3px solid transparent;
+    padding: 0.375rem 0.75rem;
+    font-size: 1rem;
+    line-height: 1.5;
+    border-radius: 0.25rem;
+    background-color: transparent;
+    color: black;
+    margin: 0.5rem;
+}
+
+.cs-alert-primary {
+    background-color: var(--alert-primary);
+    border-color: #0f71d3;
+}
+
+.cs-alert-success {
+    background-color: var(--alert-success);
+    border-color: #b7d5bd;
+}
+
+.cs-alert-danger {
+    background-color: var(--alert-danger);
+    border-color: #cdb8ba;
+}
+
+.cs-alert-warning {
+    background-color: var(--alert-warning);
+    border-color: #d1c7a8;
```

Comment:
please try to use color variables instead!!

Reply:
resolved